### DISCUSSION TITLE
how to yank from VSCode neovim to system clipboard?

### DISCUSSION THREAD

**Discussion by zwyyy456** [vscode-neovim/vscode-neovim#1179] [Q&A]
My `~/.config/nvim/init..lua` is as follow:
```lua
if vim.g.vscode then
  -- Vscode extension
  require("config.options")
else
  -- bootstrap lazy.vim, LazyVim and your plugins
  require("config.options")
end
```
And `~/.config/nvim/lua/config/options.lua` is as follows:
```lua
-- Options are automatically loaded before lazy.nvim startup
-- Default options that are always set: https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/config/options.lua
-- Add any additional options here
local opt = vim.opt -- set local variable

-- line number
opt.relativenumber = true -- 显示相对行号
opt.number = true -- 显示行号

-- 缩进
opt.tabstop = 4 -- 设置缩进为4
opt.shiftwidth = 4
opt.expandtab = true
opt.autoindent = true

-- 防止字符被包裹
opt.wrap = false

-- 高亮光标行
opt.cursorline = true

-- 启用鼠标
opt.mouse:append("a")

-- 启用系统剪贴板
opt.clipboard = "unnamedplus" -- Sync with system clipboard
opt.signcolumn = "yes" -- 左侧多一列，有利于debug和插件提示
```
I can yank from neovim to system clipboard.
The settings of vscode is as follow:
![图片](https://user-images.githubusercontent.com/45388797/226102996-2b101cdb-abe6-40ef-a54b-7bedc1030cae.png)

What should I do to enable `yank form Vscode neovim to system clipboard`?

**Comment by zwyyy456** [comment]
I have found the reason. It's because I have set InitVimPath manually, but it's a wrong path.

**Comment by c0de-intention** [comment]
Anyone else looking for a fix, the following goes in init.vim, I found it from this response:  https://github.com/vscode-neovim/vscode-neovim/issues/835#issuecomment-1073498454

>```vim
>set clipboard=unnamed
>inoremap <silent> <c-v> <Esc>:set paste<Cr>a<c-r>+<Esc>:set nopaste<Cr>a
>```
